multichain-testing/scripts/fetch-starship-chain-info.ts

```diff
@@ -27,10 +27,13 @@ const ibc: {
   data: IBCInfo[];
 } = await fetch(`${BASE_URL}ibc`).then(r => r.json());
 
-const chainInfo = await convertChainInfo({
-  chains,
-  ibcData: ibc.data,
-});
+const chainInfo = await convertChainInfo(
+  {
+    chains,
+    ibcData: ibc.data,
+  },
+  () => true,
```

Comment:
this happens to be true but should it be assumed?

Reply:
Might as well not assume. Just changed this to use the imported list of icaEnabled chains